criu/cr-service.c

```diff
@@ -781,13 +623,147 @@ static int setup_opts_from_req(int sk, CriuOpts *req)
 	if (req->has_display_stats)
 		opts.display_stats = req->display_stats;
 
-	/* Evaluate additional configuration file a second time to overwrite
-	 * all RPC settings. */
+	/* Evaluate additional configuration file (e.g., runc.conf) to overwrite all RPC settings. */
 	if (req->config_file) {
+		char *tmp_output = opts.output;
+		char *tmp_work = opts.work_dir;
+		char *tmp_imgs = opts.imgs_dir;
+
+		opts.output = NULL;
+		opts.work_dir = NULL;
+		opts.imgs_dir = NULL;
+
+		pr_debug("Would overwrite RPC settings with values from %s\n", req->config_file);
```

Comment:
Where does this line end up? Not in the configuration file set via RPC, so maybe drop that line or do that later.

Reply:
I would expect that all lines that end up in the early log buffer to have the "(00.000000)" time stamp. What is in the `runc.conf` you are using?

Reply:
```
$ cat /etc/criu/runc.conf 
log-file=/tmp/criu.log
```